wiki/Glossary/Bit_Rate/en.md

```diff
@@ -0,0 +1,11 @@
+# Bit Rate
+
+*For regulations surrounding bit rate, see: [Ranking Criteria](/wiki/Ranking_Criteria)*
+
+*Bit rate* describes the number of bits per second that can be transmitted along a digital network. Almost every music file contains something called a "bit rate". This means that a *higher bit rate* will generally mean *higher quality* (but bigger file size). Whereas, a *lower bit rate* will generally mean *lower quality* (but smaller file size).
```

Comment:
```suggestion
**Bit rate** describes the number of bits per second that can be transmitted along a digital network. Almost every music file contains something called a "bit rate". This means that a higher bit rate will generally mean *higher quality* (but bigger file size). Whereas, a *lower bit rate* will generally mean *lower quality* (but smaller file size).
```
Don't know if this really needs to be an osu! wiki article; linking to Wikipedia should be just fine imo